tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+*** Settings ***
+Library         OpenShiftCLI
+Library         OpenShiftLibrary
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+
+
+*** Test Cases ***
+Verify If Unauthorisez User Can Not Change The Permission
+    [Tags]  ODS-1661
+    ...     ODS-1660
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
+    Add OpenShift Groups To Data Science Administrators         cluster-admins  rhods-admins
+    Add OpenShift Users groups to Data Science administrators   system:authenticated
+    Save Changes In User Management Setting
+
+    Verify Expect Changes In AdminGroups Are Present In CRD     cluster-admins  rhods-admins
+    Verify Expect Changes In AllowedGroups Are Present In CRD   system:authenticated
+
+    Check User Management Option Is Available For The User     ${OCP_ADMIN_USER.USERNAME}   ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
+    Remove OpenShift Groups To Data Science Administrators     rhods-admins
+    Save Changes In User Management Setting
+    Switch Browser  1
+    Add OpenShift Groups To Data Science Administrators         rhods-noaccess
+    Add OpenShift Users Groups To Data Science Administrators   rhods-users
+
+    Verify Expect Changes In AdminGroups Are Present In CRD     cluster-admins
+    Verify Expect Changes In AllowedGroups Are Present In CRD   system:authenticated
+    Save Changes In User Management Setting
+    Page Should Contain  Unable to load User and group settings
+    Switch Browser  2
+    [Teardown]  Teardown For Admin UI   ${OCP_ADMIN_USER.USERNAME}   ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
+
+Verify User Is Able To Spawn Jupyter Notebook
+    [Documentation]     Verify Correct User Is able to Spawn Jupyter Notebook
+    [Tags]   ODS-1681
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
+    Add OpenShift Groups To Data Science Administrators         rhods-admins
+    Add OpenShift Users Groups To Data Science Administrators   rhods-admins
+    Save Changes In User Management Setting
+    Launch JupyterHub Spawner From Dashboard     ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    [Teardown]  Teardown For Admin UI  ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+Verify User Is Not Able To Spawn Jupyter Notebook
```

Comment:
could you change a bit the title to be more clear? Why user shouldn't be able to spawn? (I know what you meant, just saying it should be clear in the title)